src/vstart.sh

```diff
@@ -697,7 +697,7 @@ $extra_conf
 [mgr]
         mgr data = $CEPH_DEV_DIR/mgr.\$id
         mgr module path = $MGR_PYTHON_PATH
-        ceph daemon path = $CEPH_ROOT/src/ceph-daemon
+        ceph daemon path = $CEPH_ROOT/src/ceph-daemon/ceph-daemon
```

Comment:
and yeah, I'd live to get rid of this quirk

Reply:
But that's not relavant for this PR, or is it?

Reply:
this config entry just feels so unnecessary